tests/EVM/test_EVMABI.py

```diff
@@ -0,0 +1,40 @@
+
+import unittest
+from manticore.ethereum import ABI, EthereumError
+
+class EVMTest_ABI(unittest.TestCase):
```

Comment:
stylistically, i prefer avoiding underscores in test class names. maybe something like "EthereumAbiTests"?

Reply:
also instead of creating a new file in tests/EVM, please move this test class into test_eth.py. tests/EVM is mostly for the auto generated evm instruction tests :)